dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -1082,33 +1129,20 @@ public void shouldReturnPageHTMLForLegacyUUIDAndMultiTree() throws Exception {
      * @throws Exception
      */
     @Test
-    public void shouldReturnParserContainerUUID() throws Exception {
+    @UseDataProvider("cases")
+    public void shouldReturnParserContainerUUID(final Container container, final Template template) throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)